_tour/tour-of-scala.md

```diff
@@ -17,49 +17,49 @@ This tour contains bite-sized introductions to the most frequently used features
 of Scala. It is intended for newcomers to the language.
 
 This is just a brief tour, not a full language tutorial. If
-you want that, consider obtaining [a book](/books.html) or consulting
+you want a more detailed guide, consider obtaining [a book](/books.html) or consulting
 [other resources](/learn.html).
 
 ## What is Scala?
-Scala is a modern multi-paradigm programming language designed to express common programming patterns in a concise, elegant, and type-safe way. It smoothly integrates features of object-oriented and functional languages.
+Scala is a modern multi-paradigm programming language designed to express common programming patterns in a concise, elegant, and type-safe way. It seamlessly integrates features of object-oriented and functional languages.
 
 ## Scala is object-oriented ##
-Scala is a pure object-oriented language in the sense that [every value is an object](unified-types.html). Types and behavior of objects are described by [classes](classes.html) and [traits](traits.html). Classes are extended by subclassing and a flexible [mixin-based composition](mixin-class-composition.html) mechanism as a clean replacement for multiple inheritance.
+Scala is a pure object-oriented language in the sense that [every value is an object](unified-types.html). Types and behaviors of objects are described by [classes](classes.html) and [traits](traits.html). Classes can be extended by subclassing, and by using a flexible [mixin-based composition](mixin-class-composition.html) mechanism as a clean replacement for multiple inheritance.
 
 ## Scala is functional ##
-Scala is also a functional language in the sense that [every function is a value](unified-types.html). Scala provides a [lightweight syntax](basics.html#functions) for defining anonymous functions, it supports [higher-order functions](higher-order-functions.html), it allows functions to be [nested](nested-functions.html), and supports [currying](multiple-parameter-lists.html). Scala's [case classes](case-classes.html) and its built-in support for [pattern matching](pattern-matching.html) model algebraic types used in many functional programming languages. [Singleton objects](singleton-objects.html) provide a convenient way to group functions that aren't members of a class.
+Scala is also a functional language in the sense that [every function is a value](unified-types.html). Scala provides a [lightweight syntax](basics.html#functions) for defining anonymous functions, it supports [higher-order functions](higher-order-functions.html), it allows functions to be [nested](nested-functions.html), and it supports [currying](multiple-parameter-lists.html). Scala's [case classes](case-classes.html) and its built-in support for [pattern matching](pattern-matching.html) model algebraic types are used in many functional programming languages. [Singleton objects](singleton-objects.html) provide a convenient way to group functions that aren't members of a class.
```

Comment:
"model algebraic types" is pretty abstruse.  While you're making changes, maybe you want to say something like, "...provide the functionality of algebraic types, which are used in many functional programming languages."

Reply:
The change here is that I added in an extra "it" for "and it supports currying" but this change seems to only show up in the rich diff, unfortunately.

Reply:
I'm aware--I just think that the "model algebraic types" language (using model as a verb) is so unclear that it ought to be fixed as long as you're making broad improvements here.